packages/hooks/use-lockscreen/index.ts

```diff
@@ -37,13 +37,13 @@ export const useLockscreen = (trigger: Ref<boolean>) => {
 
   let scrollBarWidth = 0
   let withoutHiddenClass = false
-  let bodyPaddingRight = '0'
-  let computedBodyPaddingRight = 0
+  let bodyWidth = '0'
+  let computedBodyWidth = Number.parseInt(getStyle(document.body, 'width'), 10)
 
   const cleanup = () => {
     removeClass(document.body, hiddenCls.value)
     if (withoutHiddenClass) {
-      document.body.style.paddingRight = bodyPaddingRight
+      document.body.style.width = bodyWidth
```

Comment:
A fixed width is probably not a good idea. For example, after open a Dialog, the body width is fixed. At this time, the body width will not change with the width of the browser, but will change when the Dialog closed.

Reply:
Indeed, this causes it to lose its responsiveness, can `calc(100% - scrollbarWidth)` still be optimized. 
e.g. `100%` = (bodyWidth + scrollbarWidth) / document.width